### ISSUE TITLE
Show structured deploy logs output

### ISSUE THREAD

**Issue by lewish** [tada-science/dataform-tracking#12]
Currently only the contents of DBT run is shown. For large projects this is a very large, hard to parse block of text.
Should be possible to quickly see failed models, and any database exceptions that were thrown within them, rather than showing these separately.

**Comment by G2H** [comment]
Could also add deploy properties (user, models deployed..)

**Comment by G2H** [comment]
Should also be able to see the exact SQL that was run in the database. Very important when you do deletions or similar things

**Comment by lewish** [comment]
This is fixed in beaver release.
